run_scans.py

```diff
@@ -21,7 +21,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax
 
   #Build Commands
   setupCmds = []
-  preCmd = None
+  preCmd = None 
```

Comment:
there appears to be some trailing whitespace here (L24)

Reply:
Still there...

Reply:
Searching for `\s` characters in vim.  Doesn't show up:

<img width="161" alt="screen shot 2017-08-31 at 11 19 24" src="https://user-images.githubusercontent.com/6432141/29916165-436c868c-8e3e-11e7-86c4-1b9ceaef270f.png">

Reply:
It may be that changes weren't pushed to the remote...?

Reply:
Agree, current file on github appears fixed